rip/main.py

```diff
@@ -11,21 +13,17 @@
 Tl;dr"""
 
 text_response = openai.Completion.create(
-  model="text-davinci-002",
-  prompt=prompt,
-  temperature=0.7,
-  max_tokens=60,
-  top_p=1.0,
-  frequency_penalty=0.0,
-  presence_penalty=0.0
+    model="text-davinci-002",
+    prompt=prompt,
+    temperature=0.7,
+    max_tokens=60,
+    top_p=1.0,
+    frequency_penalty=0.0,
+    presence_penalty=0.0,
 )
 tldr_text = text_response.choices[0].text
 
-img_response = openai.Image.create(
-  prompt=tldr_text,
-  n=1,
-  size="1024x1024"
-)
+img_response = openai.Image.create(prompt="glacier metling in a hanging valley", n=1, size="1024x1024")
```

Comment:
img_response = openai.Image.create(prompt= tldr_text, n=1, size="1024x1024")

Reply:
yes you are totally right, I was testing with the random texts..... many thanks for your thorough review!